### PR TITLE
support for kubernetes events from agent

### PR DESCRIPTION
**Pull request by grs**

### CODE REVIEW

File: agent/lib/events.js [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        },
+        involvedObject: {
+            kind: kind,
+            name: name
+        }
+    };
+}
+
+module.exports.address_create = function (address) {
+    return event('agent', 'Create', util.format('%s %s created', address.type, address.address), 'Normal', 'Address', address.address);
```
[the remark this answers is not in the document]
Comment (grs) [reply agent/lib/events.js]: I don't see any other events that repeat the kind in the event. Seems a little redundant to me. Obviously where there are different things created/deleted with the same involved object, that does require some qualification. Not a huge issue though, so I could live with AddressCreated/AddressDeleted.

It seems odd to me to have FailedAddressCreated though, FailedAddressCreate sounds clearer. ReplicaSet seems to use SuccessfulCreate (which would also match nicely with FailedCreate). I see now you are using AddressSpaceCreateFailed. The doc you link to above is itself inconsistent on this with a mix of pre- and post- fixing with failed (even in the same table!).

Bottom line though, I don't care too much. I agree we should try to be consistent (though in general there is no real consistency with existing events). Its pretty arbitrary I think. Since you already committed, I guess it makes sense to just go with that.
